packages/desktop/views/dashboard/governance/views/ProposalDetailsView.svelte

```diff
@@ -71,13 +72,25 @@
         ]
     }
 
+    $: $selectedParticipationEventStatus, (textHintString = getTextHintString())
+
     $: isVotingDisabled =
+        !isLoaded ||
         !isProposalVotable($selectedProposal?.status) ||
-        !hasChangedAnswers(selectedAnswerValues) ||
-        hasSelectedNoAnswers(selectedAnswerValues)
+        hasSelectedNoAnswers(selectedAnswerValues) ||
+        isUpdatingVotedAnswerValues ||
+        isSelectedEqualVotedAnswers
```

Comment:
This name is just slightly unclear, maybe something like `areSelectedAndVotedAnswersEqual` or `hasSelectedNoNewAnswers` - I might be being a bit pedantic here :sweat_smile: